### PR TITLE
FWI-2162 - insights-plugin: make NewActionItemThreshold default to -1

### PR DESCRIPTION
**Pull request by jdesouza**
This PR fixes #

## Checklist
* [ ] I have signed the CLA
* [ ] I have updated/added any relevant documentation

### What alternative solution should we consider, if any?

[Internal Ticket FWI-2162](https://fairwinds.myjetbrains.com/youtrack/issue/FWI-2162)

### CODE REVIEW

File: plugins/ci/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.6.2
```
Comment (makoscafee) [inline plugins/ci/version.txt:1]: probably make this minor release... since the support for this in the backend is not released yet.

Comment (jdesouza) [reply plugins/ci/version.txt:1]: 1.7.0?

Comment (jdesouza) [reply plugins/ci/version.txt:1]: Bumped to 2.0.0

File: plugins/ci/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 1.7.0
+## 2.0.0
```
Comment (makoscafee) [inline plugins/ci/CHANGELOG.md:3]: 1.7.0 should be fine.

Comment (jdesouza) [reply plugins/ci/CHANGELOG.md:3]: I think @rbren asked for 2.0.0 because this may impact some pipelines

**Pull request merged**